src/Microsoft.VisualStudio.ProjectSystem.Managed/LinqExtensions.cs

```diff
@@ -102,5 +103,17 @@ public static bool Any<T>(this T[] array, Func<T, bool> predicate)
 
             return false;
         }
+
+        public static ImmutableArray<TValue> ToImmutableValueArray<TKey, TValue>(this Dictionary<TKey, TValue> source)
```

Comment:
Why "ImmutableValueArray" instead of just "ImmutableArray"?

Reply:
I would expect `ToImmutableArray` to return `ImmutableArray<KeyValuePair<TKey, TValue>>`.

Reply:
Oh, why isn't this just source.Values.ToImmutableArray()?

Reply:
Because that allocates an enumerator.